docs/content/users/install/docker-installation.md

```diff
@@ -22,7 +22,7 @@ You’ll need a Docker provider on your system before you can [install DDEV](dde
     colima start --cpu 4 --memory 6 --disk 100 --vm-type=qemu --mount-type=sshfs --dns=1.1.1.1
     ```
     (On macOS versions before Ventura, omit the `--vm-type=qemu` flag as it doesn't work on older OS versions.)
-    4. After [installing DDEV](ddev-installation.md), configure your system to use Mutagen—essential for DDEV with Colima—with `ddev config global --mutagen-enabled`.
+    4. After [installing DDEV](ddev-installation.md), make sure Mutagen is not disabled — essential for DDEV with Colima — with `ddev config global --performance-mode-reset`.
```

Comment:
Done

Reply:
Using spaces around em dashes is [apparently a matter of consistency](https://writing.stackexchange.com/questions/8555/em-dash-space-or-no-space-between-words), and since every other use in the docs does _not_ use spaces I’d remove them again here. (Or add them everywhere else.)

Reply:
I was *removing* the em dashes because they didn't seem useful...

Reply:
Oh no. You removed the whole line and I commented on an older one. If you could pretend this never happened I would be grateful.